requirements.txt

```diff
@@ -34,6 +34,7 @@ oauth2client==4.1.3                                         # OAuth 2.0 client l
 retrying==1.3.3                                             # general-purpose retrying library to simplify the task of adding retry behavior to just about anything
 redis==3.5.3                                                # Python client for Redis key-value store
 Flask==1.1.2                                                # Python web framework
+itsdangerous==2.0.1                                         # Flask dependency, broken support after 2.0.1 https://github.com/pallets/flask/issues/4455
```

Comment:
I thought we agree to just bump Flask to 1.1.4

Reply:
Did that, same error as before. https://github.com/rucio/rucio/actions/runs/1864534929

Reply:
The error seems different : ``` ImportError: cannot import name 'soft_unicode' from 'markupsafe' ```

Reply:
Oh, yes, you are right. https://github.com/pallets/flask/issues/4456 is the tracking issue.

Most comment suggest to not use an unsupported Flask version...
However, this would mean that we have to bump Flask by one major release, do we want to do that now?
From my POV, freezing the faulting dependency makes more sense. The pipeline would be intact afterwards and we can upgrade the Flask version some when else, when we have time to test it.

Reply:
What are your thoughts on that @rcarpa @maany ?

Reply:
Better freeze the sub-dependency in that case. I wouldn't want to do a major dependency upgrade within the release line.

Reply:
I think you will have to add it
https://github.com/rucio/rucio/blob/2930272ff394b28ed9f4d81e1b9f898d8f418951/setuputil.py#L81
too?